README.md

```diff
@@ -40,7 +38,40 @@ and the requisite tables will be created.
 
 ## Usage
 
-TODO: Write usage instructions here
+Add your Pivotal Tracker API token to your secrets.yml:
+
+`tracker_api_token: <%= ENV["PIVOTAL_TRACKER_API_TOKEN"] %>`
+
+Create a new rake task called `import.rake`
```

Comment:
If you do figure it out, but sure to namespace the tasks appropriately. `rake active_pivot:import:initial 8/12/2012`

[Here's a post about passing args to a rake task](http://thepolymathlab.com/4-ways-to-pass-arguments-to-a-rake-task). (My example would be method 3.)